packages/sites/src/domains/_lookup-portal.ts

```diff
@@ -9,7 +10,7 @@ import { IHubRequestOptions, includes } from "@esri/hub-common";
  */
 export function _lookupPortal(
   hostname: string,
-  hubRequestOptions: IHubRequestOptions
+  requestOptions: IRequestOptions
```

Comment:
Downgraded this call sig, since all this function really needs is an `IRequestOptions`